Description: These #[doc] were blocking the build of the package.
Author: Eddie Billoir <lechatp@outlook.fr>
Index: nonstick/src/lib.rs
===================================================================
--- nonstick.orig/src/lib.rs
+++ nonstick/src/lib.rs
@@ -148,8 +148,8 @@
 //! nonstick detects which implementation it should use for the current target.
 //! If you need to choose a different implementation, set the `LIBPAMSYS_IMPL`
 //! environment variable at build time. See the [`libpam_sys`] documentation.
-#![doc = ""]
-#![doc = concat!("This documentation was built for **", pam_impl_name!(), "**.")]
+//#![doc = ""]
+// #![doc = concat!("This documentation was built for **", pam_impl_name!(), "**.")]
 //!
 //! # Cargo features
 //!
